lib/koans/15_protocols.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Protocols do
+  use Koans
+
+  defprotocol School do
+    def enrol(name)
```

Comment:
Please pardon me if this is an international spelling, but I believe the correct spelling is "enroll" (with two Ls).